sdk/eventhubs/azure-messaging-eventhubs/src/main/java/com/azure/messaging/eventhubs/EventHubClientBuilder.java

```diff
@@ -630,8 +645,10 @@ private ConnectionOptions getConnectionOptions() {
             ? CbsAuthorizationType.SHARED_ACCESS_SIGNATURE
             : CbsAuthorizationType.JSON_WEB_TOKEN;
 
+        final ClientOptions options = clientOptions != null ? clientOptions : new ClientOptions();
+
         return new ConnectionOptions(fullyQualifiedNamespace, credentials, authorizationType, transport, retryOptions,
-            proxyOptions, scheduler, null);
+            proxyOptions, scheduler, options, SslDomain.VerifyMode.VERIFY_PEER_NAME);
```

Comment:
Do we plan of supporting any other mode apart from VERIFY_PEER_NAME?

Reply:
In Service Bus, they had additional configuration via environment variables. It's not a feature we ever discussed.